SwaMe/SwaMe.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can we please not include this as a DLL? I'd included it as a child project like this deliberately - the CrawDad# repo needs to be cloned next to this one

Reply:
Okay sorry about that, I misunderstood.